README.md

```diff
@@ -146,7 +159,10 @@ with multiple steps, such as a "start" state that only contains `// TODO` and a
 "final" state that contains completed implementation code.
 
 Because `state` operates on ranges of content, it is only available as
-a block command. You must pass `state` an identifier.
+a block command. You must pass `state` at _least one_ identifier, which determines
+the name of the state or states that the block belongs to. You can pass
+in a list of identifiers either through a space-separated list directly after
+the command itself, or through the `id` field of an [attribute list](attribute lists).
```

Comment:
Does this anchor linking actually work? I thought you had to do html... 
```
<a name="attribute-lists"></a>
# Attribute Lists

...
... [attribute list](#attribute-lists)
```

Reply:
It turns out that github's markdown flavor automatically creates those anchors, though you were right -- this doesn't quite work this way. I had to change `(attribute lists)` to `(#attribute-lists)`.